static/amp-form-error.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
amp-form-error.html is the page that will be opened when the amp-form returns an error.